test/utils/image/manifest.go

```diff
@@ -373,10 +373,16 @@ func ReplaceRegistryInImageURL(imageURL string) (string, error) {
 			}
 		}
 		last := strings.SplitN(parts[countParts-1], ":", 2)
+		var version string
+		if len(last) == 1 {
+			version = "latest"
```

Comment:
Can we not just fix this and make the error be fail fast?    It seems wierd that we'd allow pull specs without tags anywhere.